rcl/test/rcl/test_graph.cpp

```diff
@@ -706,16 +706,104 @@ TEST_F(
   rcl_reset_error();
 }
 
+/* Test the rcl_wait_for_publishers function.
+ */
+TEST_F(
+  CLASSNAME(TestGraphFixture, RMW_IMPLEMENTATION),
+  test_rcl_wait_for_publishers
+) {
+  rcl_ret_t ret;
+  rcl_node_t zero_node = rcl_get_zero_initialized_node();
+  rcl_allocator_t zero_allocator = static_cast<rcl_allocator_t>(
+    rcutils_get_zero_initialized_allocator());
+  rcl_allocator_t allocator = rcl_get_default_allocator();
+  const char * topic_name = "/topic_test_rcl_wait_for_publishers";
```

Comment:
I tried to use this API and I get a `invalid topic name` return code if the topic isn't absolute.